src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/GetDelegateForFunctionPointerTests.cs

```diff
@@ -48,7 +48,7 @@ public void GetDelegateForFunctionPointer_CollectibleType_ReturnsExpected()
 
             Delegate functionDelegate = Marshal.GetDelegateForFunctionPointer(ptr, type);
             VerifyDelegate(functionDelegate, targetMethod);
-            GC.KeepAlive(type);
+            GC.KeepAlive(functionDelegate);
```

Comment:
You need to keep alive the "d" not the "functionDelegate"

Reply:
oh.. my fault. let me update all of these and update this PR